MAUI/PDF-Viewer/Free-Text.md

```diff
@@ -124,6 +124,22 @@ void CustomizeDefaultFreeTextSettings()
 {% endhighlight %}
 {% endtabs %}
 
+### Hide the Border of a Free Text Annotation
```

Comment:
As we discussed change the title as how to create borderless free text annotation and modify its description accordingly